src/rocksdb/src/lib.rs

```diff
@@ -655,13 +641,13 @@ fn rocksdb_core_loop<K, V, M, O, IM, F>(
         });
 
     let db: DB = match retry_result {
-        Ok(db) => {
-            drop(creation_error_tx);
-            db
-        }
+        Ok(db) => db,
         Err(e) => {
-            // Communicate the error back to `new`.
-            let _ = creation_error_tx.send(e);
+            tracing::error!(
+                "failed to create rocksdb at {}: {}",
```

Comment:
This will make it to Sentry. Is this a rare/interesting enough error that we should be notified if it happens?

Reply:
I think so, yes. We have a bunch of errors in Senry which are mostly user errors.